playbook/staging-guide.yml

```diff
@@ -0,0 +1,38 @@
+# Using https://github.com/nickjj/ansible-docker/ to install docker
```

Comment:
The header should say what the playbook does. (Installing Docker is not its main purpose.)